ui/src/components/filter/composables/useFilters.ts

```diff
@@ -20,28 +20,25 @@ const filterItems = (items: object[], element: object) => {
     return items.filter((item) => compare(item, element));
 };
 
-const buildComparator = (value: string, multiple = false): Comparator => {
-    return {label: value, value, multiple};
+const buildComparator = (label: string, value: string, multiple: boolean = false): Comparator => {
+    return {label, value, multiple};
 };
 
 export function useFilters(prefix: string) {
     const {t} = useI18n({useScope: "global"});
 
     const comparator = (which: string) => `filters.comparators.${which}`;
-
     const COMPARATORS: Record<string, Comparator> = {
-        IS: buildComparator(t(comparator("is"))),
-        IS_ONE_OF: buildComparator(t(comparator("is_one_of")), true),
-
-        IS_NOT: buildComparator(t(comparator("is_not"))),
-        IS_NOT_ONE_OF: buildComparator(t(comparator("is_not_one_off")), true),
-
-        CONTAINS: buildComparator(t(comparator("contains")), true),
-        NOT_CONTAINS: buildComparator(t(comparator("not_contains")), true),
-
-        IN: buildComparator(t(comparator("in"))),
-        BETWEEN: buildComparator(t(comparator("between"))),
-        STARTS_WITH: buildComparator(t(comparator("starts_with"))),
+        EQUALS: buildComparator(t(comparator("is")), "$eq"),
```

Comment:
Potentially, might be a good idea to uniform the expressions - to not show label `IS` for `EQUALS`, but the `EQUALS` itself.

Although, this is more of a question for a product team, so pinging @Ben8t here for an opinion.

Reply:
Not sure to understand: does it shows "IS" here? Can you give me a bit more context please?

Reply:
It's the comparator chooser of filters:

![image](https://github.com/user-attachments/assets/b8557b30-038f-4c99-b896-02fa087949dd)

For example, what is right now `IS` is going to be sent to backend as `EQUALS` when this is merged. Do we want to align the wording on that, so it's not labeled as `IS` anymore, but as `EQUALS`?